calico/felix/devices.py

```diff
@@ -417,11 +417,14 @@ def watch_interfaces(self):
                         _log.debug("IFLA_OPERSTATE: %s", operstate)
 
                 if (ifname and
-                    ifname in if_last_flags and
-                    (msg_type == RTM_DELLINK or operstate != IF_OPER_UP)):
+                        ifname in if_last_flags and
+                        (msg_type == RTM_DELLINK or operstate != IF_OPER_UP)):
```

Comment:
PEP-8, avoids aligning the lines of the expression with the block introduced by the `if:`.